app/services/sendspin_service.py

```
@@ -0,0 +1,838 @@
+            self._notify_clients_changed()
+
+    def _setup_client(self, client: SendspinClient) -> None:
+        if not self._group:
+            self._group = client.group
+            self._setup_group(self._group)
+        elif client.group != self._group:
+            try:
+                self._loop and self._loop.call_soon_threadsafe(
+                    lambda: asyncio.ensure_future(self._group.add_client(client))
+                )
+            except Exception:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)